src/improvements/tasks/MultisigTask.sol

```diff
@@ -143,6 +143,10 @@ abstract contract MultisigTask is Test, Script {
         validate(accountAccesses, actions);
         print(actions, accountAccesses, optionalChildMultisig, true);
 
+        if (optionalChildMultisig != address(0)) {
```

Comment:
I think this logic is better here incase a nested safe ever uses a different entrypoint that `signFromChildMultisig` to simulate.

Reply:
It's not used anywhere else for now so I'll leave it as is.